theme/static/css/hydroshare_core.css

```
@@ -4016,11 +4019,20 @@ ul#fb-files-container .hs-upload-indicator {
 	font-size: 12px;
 	cursor: default;
 }
+.fb-file-name-display {
+	color: #555;
+	font-size: 12px;
+	cursor: default;
+}
 
 .fb-view-list .fb-file-name {
 	font-size: 14px;
 	white-space: nowrap;
 }
+.fb-view-list .fb-file-name-display {
+	font-size: 14px;
+	white-space: nowrap;
+}
```

I recommend using selector lists for all these duplicate selectors:
https://developer.mozilla.org/en-US/docs/Learn_web_development/Core/Styling_basics/Basic_selectors#selector_lists
```suggestion
.fb-view-list .fb-file-name, .fb-view-list .fb-file-name-display {
	font-size: 14px;
	white-space: nowrap;
}
```
This avoids the duplication, so that if we ever wanted to modify in the future, we would modify in just one location instead of 2